docs/monitoring-usage.rst

```diff
@@ -154,6 +154,85 @@ before you call
     projects.metricDescriptors
 
 
+Groups
+------
+
+A group is a dynamic collection of *monitored resources* whose membership is
+defined by a `filter`_.  These groups are usually created via the `Stackdriver
+dashboard`_. You can list all the groups in a project with the
+:meth:`~gcloud.monitoring.client.Client.list_groups` method::
+
+    >>> for group in client.list_groups():
+    ...     print(group.id, group.display_name, group.parent_id)
+    ('1001', 'Production', '')
+    ('1002', 'Front-end', '1001')
+    ('1003', 'Back-end', '1001')
```

Comment:
Though an int conversion works in this example, it does not make sense in a general case, and could raise Errors for someone copy-pasting the sample code.

I will update the example to have at least one non-numeric ID.